.reuse/dep5

```diff
@@ -0,0 +1,49 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: ansys-templates
+Upstream-Contact: pyansys.support@ansys.com
+Source: https://github.com/ansys/ansys-templates
+
+
+Files: src/ansys/templates/python/*
+Copyright: 2023 ANSYS, Inc.
```

Comment:
Can't the year be made dynamic somehow? If we have to change it every single year it is going to be a pain...

Reply:
This tool needs to be executed again with the desired year. For example, if you run previous command with:

```
reuse annotate --year 2024 ...
```

It will update the "copyright" by generating something like:

```diff
-Copyright: 2023 ANSYS, Inc.
+Copyright: 2023-2024 ANSYS, Inc.
```

Reply:
We could add an action for this to forget about it.

Reply:
Yeah, definitely.. We should try to automate that as much as possible. Otherwise it is going to be a nightmare.